src/components/ChatFirstLoading/ChatFirstLoadingStyled.js

```diff
@@ -0,0 +1,27 @@
+import styled from 'styled-components';
+
+export const ChatStyled = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  min-height: 100vh;
```

Comment:
This rule will afect the whole page height and add vertical scrol

Reply:
I'll chage it when I'll started to add styles to this page.